src/components/shared/Entity/Header.tsx

```diff
@@ -32,17 +30,6 @@ interface Props {
 
 export const buttonSx: SxProps<Theme> = { ml: 1 };
 
-const stickySx: SxProps<Theme> = {
```

Comment:
Not related to logs - but don't need the sticky header anymore since we have a Topbar